MAUI/DataGrid/loadMore.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: Load More in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Load More support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Load More in MAUI DataGrid (SfDataGrid)
+
+The DataGrid enables the LoadMore option when the `SfDataGrid.AllowLoadMore` property is set to `true`, and also by setting the SfDataGrid.`LoadMoreCommand` property. When LoadMore is enabled, the control loads a subset of data to its data source at runtime using DataGridLoadMoreView.
+
+When the grid reaches maximum offset while scrolling down, an interactive DataGridLoadMoreView is displayed. Tapping the load more view triggers a command to add more data to the data source of the grid at runtime.
```

Comment:
The DataGrid enables the "Load More" option when the `SfDataGrid.AllowLoadMore` property is set to true. When the "Load More" feature is enabled, an interactive load more view will be displayed on the `SfDataGrid` only when the grid reaches the maximum scroll offset while scrolling down. It loads a subset of data into its data source at runtime when users tap the `DataGridLoadMoreView`.